db/migrate/20230203073707_create_comment.rb

```diff
@@ -0,0 +1,11 @@
+class CreateComment < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.text :text
+      t.integer :comments_counter, default: 0
+      t.references :author, foreign_key: { to_table: :users }
+      t.references :post, foreign_key: { to_table: :posts }
```

Comment:
- [ ] **[OPTIONAL]** I think that you can use a "true" value for the post foreign key instead of using the `to_table` helper. This is because Rails by convention takes the name of the column `post` and references the plural of it as the foreign key table, so there's no need to specify it! 🤓 🔧 

```suggestion
      t.references :post, foreign_key: true
```